sklearn_bench/dbscan.py

```diff
@@ -35,7 +35,7 @@ def main():
     #  'kdtree' when running unpatched scikit-learn.
 
     # Time fit
-    time, _ = bench.measure_function_time(dbscan.fit, X, params=params)
+    time, _ = bench.measure_function_time(dbscan.fit, X, params=params, stage='fit')
```

Comment:
Try to align `stage` and `stages` arguments of `bench.measure_function_time` and `bench.print_output` if possible